docs/internal/admin/List-Core-Actions.md

```diff
@@ -0,0 +1,288 @@
+# List of Core Actions
+
+The snapshot loaded with the system at the start called `core_templates` has a list of all core-actions included in template `Sample template`.
+
+## Disclaimer
+
+The plugin actions that will be listed here are different from actions that users can act in the sytem, this are only meant for admins and configure how the system will work.
+Core-actions aren't meant to be modified, and only someone with expertise should be dealing with these. Otherwise some of the features that rely on certain actions would stop working.
+
+## Briefing
+
+Core-actions are listed in each template and are configurations that run after certain events happen triggered by changes done by users in Applications linked to the template.
+These actions should be added at the start of a template creation - one siimple way is to duplicate the `Sample template` to start with all of these core-actions in the new template configuration.
```

Comment:
```suggestion
These actions should be added at the start of a template creation - one simple way is to duplicate the `Sample template` to start with all of these core-actions in the new template configuration, although it is recommended you duplicate one of the other core templates that mostly closely matches your requirements, and it will also have all the core actions included (with some exceptions for a couple of the simpler templates)
```